src/main/java/bio/terra/janitor/service/cleanup/StairwayLoggingHooks.java

```diff
@@ -0,0 +1,74 @@
+package bio.terra.janitor.service.cleanup;
+
+import bio.terra.janitor.db.TrackedResourceId;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.HookAction;
+import bio.terra.stairway.StairwayHook;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
+
+public class StairwayLoggingHooks implements StairwayHook {
```

Comment:
How about `CleanupLoggingHook`? at least `Hook` instead of `Hooks` to match interface.
Add a javadoc comment for the class.

Reply:
Done